packages/web-client/app/css/card-pay/balances.css

```diff
@@ -1,3 +1,49 @@
+.card-pay-dashboard__container {
+  display: flex;
+  justify-content: center;
+}
+
+.card-pay-dashboard__balances {
+  padding: var(--boxel-sp-xxl) var(--boxel-sp-xl);
+}
+
+.card-pay-dashboard__balances-title {
+  margin-bottom: var(--boxel-sp-xxl);
+  font-weight: normal;
+}
+
+.card-pay-dashboard__balances-section {
+  max-width: 72rem;
+  border-top: 1px solid #5f5d71;
+  padding-top: var(--boxel-sp);
+}
+
+.card-pay-dashboard__balances-section-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card-pay-dashboard__balances-section-header-button-icon {
+  margin-left: var(--boxel-sp-xs);
+}
+
+.card-pay-dashboard__balances-section-title {
+  font-weight: normal;
+}
+
+.card-pay-dashboard__balances-count {
+  padding-left: var(--boxel-sp-xxs);
+  color: var(--boxel-cyan);
+}
+
+.card-pay-dashboard__balances-cards {
+  display: grid;
+  gap: var(--boxel-sp); /* FIXME cards seem to have an extra right-side gap? */
```

Comment:
The gap is dependent on the container width; you can see in this GIF that the column and row gaps are the same at the beginning but the column gaps grow as the container gets wider. I couldn’t find a way around this, is it okay?

![grid-gap-screencast 2021-07-27 12-39-47](https://user-images.githubusercontent.com/43280/127194111-dbc36fdd-b823-440f-a16d-31c98d8ffbbc.gif)

Reply:
Another option: I like using the `auto-fill` type grid (example: https://boxel.stack.cards/cards-balances-cs-452/#/scenarios/media-registry/bunny_records), where it wraps if the screen gets smaller. Since the cards have specific widths, you do something like: `grid-template-columns: repeat(auto-fill, 22.75rem);` and then restrict the max-width of the container.

Reply:
thanks for the auto-fill suggestion. I have sadness about how it makes the right edge not align with the button, but I’m just going to leave it for now, if we get more guidance about how to handle responsiveness we can return to it.

![autofill-screencast 2021-07-29 09-40-39](https://user-images.githubusercontent.com/43280/127531567-18df908b-874c-45b9-8290-4ab79912e26c.gif)